scss/bootstrap/components/civicase/_notification-badge.scss

```diff
@@ -0,0 +1,52 @@
+.crm_notification-badge {
+  background: $brand-primary;
+  border-radius: $border-radius-base;
+  color: $crm-white;
+  display: inline-block;
+  font-size: $font-size-base;
+  line-height: 14px;
+  margin: 0 2px;
```

Comment:
why is this margin necessary? A component shouldn't usually define the margin around it, that's the job of its parent